resolver/resolver.cc

```diff
@@ -547,7 +565,7 @@ class ResolveConstantsWalk {
     }
 
 public:
-    ResolveConstantsWalk(core::Context ctx) : nesting_(make_unique<Nesting>(nullptr, core::Symbols::root())) {}
+    ResolveConstantsWalk(core::Context ctx) : nesting_(nullptr) {}
```

Comment:
There's already going to be a `ClassDef ::<root>` around all these
definitions, which means we'll hit a `preTransformClassDef` visitor that
will push `::<root>` into the nesting list when the traversal reaches
there.